src/components/GameCard.tsx

```diff
@@ -9,17 +9,71 @@ interface GameCardProps {
     gameTitle: string;
     gameImage: any;
     gameDescription: string;
+    gamePlatforms: Array<string>;
+    gameGenres: Array<string>;
+    gameUniverses: Array<string>;
```

Comment:
Are we going to want this required for every game? What is the universe for something like Chicory, for example?

Reply:
This is a good point. I actually just checked and Chicory specifically returns empty, so we probably do want this to be optional.
Created bug for milestone 3: https://gaming-backlog.atlassian.net/browse/GB-72